project-base/storefront/utils/ui/useClickClosePopup.ts

```
@@ -0,0 +1,23 @@
+import { useEffect } from 'react';
+
+const useClickClosePopup = (refs: React.RefObject<HTMLElement>[], onOutsideClick: () => void) => {
+    useEffect(() => {
+        const handleDocumentClick = (event: MouseEvent) => {
+            const isClickedInsideRefs = refs.some((ref) => {
+                return ref.current && ref.current.contains(event.target as Node);
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)